pkg/nordic_softdevice_ble/src/gnrc_nordic_ble_6lowpan.c

```diff
@@ -63,6 +63,26 @@
 
 #define BLE_PRIO                    (GNRC_NETIF_PRIO)
 
+/* XXX: netdev required by gnrc_netif, but not implemented fully for
+ * nordic_softdevice_ble for legacy reasons */
+static int _netdev_init(netdev_t *netdev);
+static int _netdev_get(netdev_t *netdev, netopt_t opt,
+                       void *value, size_t max_len);
+static int _netdev_set(netdev_t *netdev, netopt_t opt,
+                       const void *value, size_t value_len);
+
+static const netdev_driver_t _ble_netdev_driver = {
```

Comment:
I suggest moving this to the bottom of the file and then remove the static declarations above (if not needed anymore).